stonesoup/types/state.py

```diff
@@ -20,12 +20,12 @@ class State(Type):
     Most simple state type, which only has time and a state vector."""
     timestamp: datetime.datetime = Property(
         default=None, doc="Timestamp of the state. Default None.")
-    state_vector: StateVector = Property(doc='State vector.')
+    state_vector: Union[StateVector, StateVectors] = Property(doc='State vector.')
```

Comment:
I think this should be reverted, but then redefined `state_vector` on `ParticleState`.